polkadot-parachain/src/chain_spec/penpal.rs

```diff
@@ -14,10 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
-use crate::chain_spec::{get_account_id_from_seed, Extensions};
+use crate::chain_spec::{
+	get_account_id_from_seed, get_collator_keys_from_seed, Extensions, SAFE_XCM_VERSION,
+};
 use cumulus_primitives_core::ParaId;
-// use rococo_parachain_runtime::{AuraId};
-use crate::chain_spec::{get_collator_keys_from_seed, SAFE_XCM_VERSION};
+use parachains_common::{AccountId, AuraId};
```

Comment:
This is fine. Just watch out because AuraId is different for statemint than all the rest (for historical reasons).

Reply:
yes, thank you, I saw that StatemintAuraId before